vickev/howdypix
# Refactor server schema files

## Description

Refactoring of the GraphQL schema into modules. Also fixed linting in the server.





## How to test?

- Check-out this branch.
- Run the project, it should work.

## Checklist
